config/config-gc.yaml

```diff
@@ -0,0 +1,29 @@
+# Copyright 2018 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-gc
+  namespace: knative-serving
+data:
+  # Delay after revision creation before considering it for GC
+  stale-revision-create-delay: "24h"
+  # Duration since a route has been pointed at a revision before it should be GC'd
+  # This must be longer than the controller resync period
+  stale-revision-timeout: "15h"
+  # Minimum number of generations of revisions to keep before considering for GC
+  stale-revision-minimum-generations: "1"
+  # Minimum staleness duration before updating lastPinned
+  stale-revision-lastpinned-debounce: "5h"
```

Comment:
I would rephrase this: `To avoid constant updates, we allow an existing annotation to be stale by this amount before we update the timestamp`.

Reply:
done